example_test.go

```diff
@@ -1,10 +1,8 @@
-package paypal_test
-
-import paypal "github.com/plutov/paypal"
+package paypal
 
 func Example() {
 	// Initialize client
-	c, err := paypal.NewClient("clientID", "secretID", paypal.APIBaseSandBox)
+	c, err := NewClient("clientID", "secretID", APIBaseSandBox)
```

Comment:
Since it's an example, this function shouldn't be exposed in the `paypal` package, but should be visible only in godoc.